pkg/scheduler/apis/config/v1beta2/default_plugins_test.go

```diff
@@ -114,90 +113,6 @@ func TestApplyFeatureGates(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "DefaultPodTopologySpread disabled",
```

Comment:
keep this test but change to another plugin, like `JobTrackingWithFinalizers`?

Reply:
That's not a plugin. That's a feature for the job controller.

But there should be a test for `VolumeCapacityPriority`

Reply:
I can create a separate PR to add more test for `VolumeCapacityPriority`.

Reply:
sorry for wrong words, but shouldn't we change to a beta feature, and verify whether disable this feature-gate works? `VolumeCapacityPriority` is alpha now and disabled by default.

Reply:
unit tests are independent of whether the feature is alpha or beta. The test itself sets the value of the gate, regardless of the graduation status.